src/common/gui/option.cpp

```diff
@@ -1186,21 +1266,12 @@ void OptionBox::_DrawCursor(const OptionCellBounds &bounds, float scroll_offset,
     // The Draw() function (and all helper functions) should be able able to
     // render without knowledge of the private member variable _scroll_offset.
 
-    float x, y;
-
-    // Should never scissor the cursor
-    VideoManager->DisableScissoring();
-
-    float cursor_offset = 0.0f;
-
-    // [phuedx] The scroll_offset has already been calculated and projected on to the current coordinate system
-    if(_scrolling) {
-        cursor_offset = -scroll_offset;
-    }
+    float x = 0.0f;
```

Comment:
I chose to scroll the cursor while the menu is scrolling.  I felt that it looks better this way.